### PR TITLE
Update banner

### PR DESCRIPTION
**Pull request by pascal22p**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -352,6 +352,17 @@ feature {
     maxTps = 10
     timeoutInSec = 5
   }
+
+  banner {
+    home {
+      enabled = true
+      heading.en = "Help improve GOV.UK"
```
Comment (Sachin300793) [inline conf/application.conf:359]: Can we put heading text in Messages ?

Comment (Sachin300793) [inline conf/application.conf:359]: Can we put heading text in Message or any specific reason ?

Comment (pascal22p) [reply conf/application.conf:359]: If you put the text in message then it is part of the codebase which I don't want. The text should be configurable at runtime so via app-config-*.
Which means no code change, no  lengthy QA check to push a new banner except a design check is staging. It speeds up massively the change of the banner.

Comment (Sachin300793) [reply conf/application.conf:359]: understood , great

**Review by Sachin300793 — APPROVED** [review]

**Review by Luka-Abey — APPROVED** [review]

**Pull request merged**